src/components/ReadingProgressPage/ReadingHistory/AddReading/index.tsx

```diff
@@ -0,0 +1,322 @@
+/* eslint-disable max-lines */
```

Comment:
This component needs to be broken down into smaller ones. It's getting pretty big